sanic/sanic.py

```diff
@@ -247,7 +243,7 @@ def converted_response_type(self, response):
     def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
             after_start=None, before_stop=None, after_stop=None, sock=None,
             workers=1, loop=None, protocol=HttpProtocol, backlog=100,
-            stop_event=None):
+            stop_event=None, logger=None):
```

Comment:
I guess I'm confused about this change overall as well. Without the boolean doing anything how is someone supposed ot override logging settings?